client/Dockerfile

```diff
@@ -0,0 +1,8 @@
+from nginx
+
+maintainer eLife Reviwer Product Team <reviewer-product@elifesciences.org>
+
+copy deployment/nginx.conf /etc/nginx/nginx.conf
```

Comment:
isn't the convention for these to all be uppercase (`COPY`)?

Reply:
Yep